### PR TITLE
docs: Add Azure DevOps Server & Services to integrations readme

### PR DESCRIPTION
**Pull request by barryadk**
* Adding Azure DevOps Server section to README
* Adding Azure DevOps Services section to README
* Adding basic example json configs for each
* Slight reordering of settings in the docs all integrations

### CODE REVIEW

File: examples/integrations/README.md [hunk 1]
```diff
@@ -42,11 +42,40 @@ Integrations are created for an API using the `integration:create` command. The
...
+* **branch**: The branch to synchronize. If it does not exist, it will be created based on your default branch. Must not contain whitespace characters.
 * **syncMethod**: "Basic Sync" or "Advanced Sync". See [*Property: `syncMethod`*](#property-syncmethod) for details.
+* **target**: The type of code to generate and push to the repository. See [*Property: `target`*](#property-target) for details.
+* **outputFolder**: The output folder for the generated code or definition.
+* **outputFile**: If target is the YAML/JSON definiton, this is the filename for the generated definition.
+* **enabled**: Enables the integration, if set to `false` the integration will be saved but will not execute. Default value is `true`.
+
+## Azure DevOps Server Integration
+* **name**: Name of the integration
+* **configType**: "AZURE_DEVOPS_SERVER" is used to create a Azure DevOps Server integration.
```
Comment (hkosova) [inline examples/integrations/README.md:54]: a Azure -> an Azure?

File: examples/integrations/README.md [hunk 2]
```diff
@@ -42,11 +42,40 @@ Integrations are created for an API using the `integration:create` command. The
...
 * **branch**: The branch to synchronize. If it does not exist, it will be created based on your default branch. Must not contain whitespace characters.
+* **syncMethod**: "Basic Sync" or "Advanced Sync". See [*Property: `syncMethod`*](#property-syncmethod) for details.
 * **target**: The type of code to generate and push to the repository. See [*Property: `target`*](#property-target) for details.
+* **outputFolder**: The output folder for the generated code or definition.
 * **outputFile**: If target is the YAML/JSON definiton, this is the filename for the generated definition.
+* **enabled**: Enables the integration, if set to `false` the integration will be saved but will not execute. Default value is `true`.
+
+## Azure DevOps Services Integration
+* **name**: Name of the integration
+* **configType**: "AZURE_DEVOPS_SERVICES" is used to create a Azure DevOps Service integration.
```
Comment (hkosova) [inline examples/integrations/README.md:69]: a Azure -> an Azure?

File: examples/integrations/README.md [hunk 3]
```diff
@@ -42,11 +42,40 @@ Integrations are created for an API using the `integration:create` command. The
...
+* **outputFolder**: The output folder for the generated code or definition.
+* **outputFile**: If target is the YAML/JSON definiton, this is the filename for the generated definition.
+* **enabled**: Enables the integration, if set to `false` the integration will be saved but will not execute. Default value is `true`.
+
+## Azure DevOps Server Integration
+* **name**: Name of the integration
+* **configType**: "AZURE_DEVOPS_SERVER" is used to create a Azure DevOps Server integration.
+* **url**: Azure DevOps Server host, typically `http(s)://server[:port]/tfs`
+* **personalAccessToken**: Personal access token for accessing the repository. The token must have _Code (read and write)_ scope.
+* **projectCollection**: Project collection which contains the target repositories project.
```
Comment (hkosova) [inline examples/integrations/README.md:57]: "repositories project" looks like a typo.

File: examples/integrations/azure_devops_server.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Azure DevOps Server Sync",
+    "configType": "AZURE_DEVOPS_SERVER",
+    "url": "https://azure-devops.onprem.swaggerhub.com",
```
Comment (hkosova) [inline examples/integrations/azure_devops_server.json:4]: Consider using an `example.com` domain instead. Also, I think default installations have `/tfs` in their base path / API root.
```suggestion
    "url": "https://azure-devops.example.com/tfs",
```

**Review by hkosova — CHANGES_REQUESTED** [review]

**Comment by barryadk** [comment]
Thanks for your feedback @hkosova

**Review by conor-ward — APPROVED** [review]
Looks good, only thinking on this now. Should we add something about Azure DevOps Server only being available to On Premise. Should probably be mentioned on the API itself too.

**Pull request merged**